start/sw/web-app/django.md

```diff
@@ -10,130 +10,103 @@ Django is a high-level Python Web framework that encourages rapid development an
 
-For Python:
-
-```
-sudo dnf install python-pip
-```
-
-or Python 3:
-
 ```
-sudo dnf install python3-pip
+$ sudo dnf install python3-pip
 ```
 
-# Get started
+## Get started
 
-## How to install virtualenv
+### How to install virtualenv
 
 It's recommended to keep your project inside virtual environment. But you can skip this step if you want. 
 Enter this command to your command line for installation:
 
 ```
-sudo dnf install python-virtualenv
-```
-
-or installation for Python3:
-
-```
-sudo dnf install python3-virtualenv
+$ sudo dnf install python3-virtualenv
 ```
 
 
 Get inside your working directory or create new where you want to store your virtual environment:
 
 ```
-mkdir working-dir
-cd working-dir
+$ mkdir working-dir
+$ cd working-dir
 ```
 
 Now we will create virtual environment called venv inside this directory using following command:
 
 ```
-virtualenv venv
+$ virtualenv venv
 ```
 
 When it's done, we will activate our virtualenv. Now we will use following commands:
 
 ```
-cd venv
-source bin/activate
+$ cd venv
+$ source bin/activate
 ```
 
 When it's activated we will see indicator of same name as our virtualenv. 
 Now we will check if we really have empty environment:
 
 ```
-pip freeze
+$ pip freeze
 ```
 
 It will return nothing.
 
-## How to install Django
+### How to install Django
 
 Now if you installed `virtualenv` do not leave it, we will install Django now. If you installed `virtualenv` use this command 
 inside your working directory with activated `virtualenv`:
 
 ```
-pip install django
+$ pip3 install django
 ```
 
 If you are following a book or a tutorial, you might want to specify the same exact version like this
 
 ```
-pip install django==1.10
-```
-
-For Python 3:
-
-```
-pip3 install django==1.10
+$ pip3 install django==1.10
 ```
 
 
 Of course you can use the version that is prepared by Fedora community by typing:
 
-For Python 2:
-
 ```
-sudo dnf install python-django
-```
-
-or Python 3:
-
-```
-sudo dnf install python3-django
+$ sudo dnf install python3-django
 ```
 
 and when it's finished use following command to start your first project called mysite:
 
 ```
-django-admin startproject mysite
+$ django-admin startproject mysite
 ```
 
 Now if you will check your directory using ls you will see new directory called mysite. We 
 will go inside that directory and we will run our project by following commands:
 
 ```
-cd mysite/
-python manage.py runserver
+$ cd mysite/
+$ python3 manage.py runserver
 ```
 
 You will see message with information about your development server. When you insert URL into your browser, 
 you will see welcome page (default URL for development server is http://127.0.0.1:8000/).
 And you successfully started your first Django project.
 
-## Switching Virtualenvs
+### Switching Virtualenvs
 
 You can use `virtualenvwrapper` after installing it with usual command
+
 ```
-dnf install python-virtualenvwrapper
+$ sudo dnf install python3-virtualenvwrapper
```

Comment:
This is non django-related too, but rather general virtualenv usage, right?

Reply:
exactly, also a very specific way to do things. upstream recommends pipenv instead (we are working on packaging it into fedora and will update general python pages to mention pipenv)

Reply:
Ok, wouldn't removing that line make sense then?